Don't print AST if-clauses when emitting SIL.

eeckstein opened the pull request:
This crashed while emitting the SIL of the stdlib build.

eeckstein commented:
@swift-ci smoke test and merge

jrose-apple commented:
We can't print un-type-checked ASTs? cc @akyrtzi

akyrtzi commented:
I've discussed with Erik, to fix SIL printing he should just add

```
result.PrintIfConfig = false;
```

to:

```
  /// Retrieve the set of options suitable for printing SIL functions.
  static PrintOptions printSIL() {
```

I'll open another PR to fix the crash. This PR as is now should not be merged.

eeckstein committed:
Don't print AST if-clauses when emitting SIL.

It's not required and also this crashed while emitting the SIL of the stdlib build.

eeckstein commented:
@swift-ci smoke test and merge
